backend/group_service/group/management/commands/runservice.py

```diff
@@ -0,0 +1,91 @@
+import asyncio
+import signal
+import functools
+
+from django.core.management import BaseCommand
+
+from backend.group_service.group.infra.adapter.group_create_command_handler \
+    import GroupCreateCommandHandler
+from backend.group_service.group.infra.adapter.group_update_command_handler \
+    import GroupUpdateCommandHandler
+from backend.group_service.group.app.group_application_service \
+    import GroupApplicationService
+from backend.common.messaging.infra.redis.redis_message_subscriber \
+    import RedisMessageSubscriber
+from backend.common.command.group_create_command \
+    import GROUP_CREATE_COMMAND
+from backend.common.command.group_update_command \
+    import GROUP_UPDATE_COMMAND
+from backend.common.rpc.infra.adapter.redis.redis_rpc_server \
+    import RedisRpcServer
+
+
+class Command(BaseCommand):
+
+    group_application_service = GroupApplicationService()
+    group_create_command_handler = GroupCreateCommandHandler(
+        group_application_service=group_application_service)
+    group_update_command_handler = GroupUpdateCommandHandler(
+        group_application_service=group_application_service)
+    subscriber = RedisMessageSubscriber()
+    rpc_server = RedisRpcServer()
+
+    def register_signal_handler(self, loop):
+        for signame in ('SIGINT', 'SIGTERM'):
+            loop.add_signal_handler(
+                getattr(signal, signame),
+                functools.partial(asyncio.ensure_future,
+                                  self.shutdown(signame, loop)))
```

Comment:
`register_signal_handler` defined in common/utils/signal. This code should be removed and refactored